shared_model/interfaces/visitor_apply_for_all.hpp

```diff
@@ -19,19 +19,56 @@
 #define IROHA_VISITOR_APPLY_FOR_ALL_HPP
 
 #include <boost/variant/static_visitor.hpp>
+#include <type_traits>
 
 namespace shared_model {
   namespace detail {
 
+/**
+ * Macro for defining concept check of the member function
+ */
+#define DEF_HAS_FUNC(FUNC_NAME)                                          \
```

Comment:
What is the reason to use it? If a class doesn't have `toString` or some required method it will lead compilation to fail, isn't it?

Reply:
If Reasons in ErrorResponse use interfaces, this is not needed.